mock-abis/src/main/java/io/mosip/proxy/abis/controller/ProxyAbisConfigController.java

```diff
@@ -17,186 +17,197 @@
+	@Operation(summary = "Get cached biometrics by hash", description = "Get cached biometrics by hash", tags = {
+			"Proxy Abis config API" })
+	@ApiResponses(value = { @ApiResponse(responseCode = "200", description = "OK"),
+			@ApiResponse(responseCode = "201", description = "Created", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "403", description = "Forbidden", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(hidden = true))) })
+	public ResponseEntity<List<String>> getCacheByHash(@PathVariable String hash) throws Exception {
+		logger.info("Get cached biometrics by hash: " + hash);
+		try {
+			return new ResponseEntity<>(proxyAbisConfigService.getCachedBiometric(hash), HttpStatus.OK);
+		} catch (RuntimeException exp) {
+			logger.error("Exception in cache request: " + exp.getMessage());
+			throw exp;
+		}
+	}
+
+	@RequestMapping(value = "cache", method = RequestMethod.DELETE)
+	@Operation(summary = "Delete cached biometrics", description = "Delete cached biometrics", tags = {
+			"Proxy Abis config API" })
+	@ApiResponses(value = { @ApiResponse(responseCode = "200", description = "OK"),
+			@ApiResponse(responseCode = "201", description = "Created", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "401", description = "Unauthorized", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "403", description = "Forbidden", content = @Content(schema = @Schema(hidden = true))),
+			@ApiResponse(responseCode = "404", description = "Not Found", content = @Content(schema = @Schema(hidden = true))) })
+	public ResponseEntity<String> deleteCache() throws Exception {
+		logger.info("Delete cached biometrics Request");
+		try {
+			proxyAbisConfigService.deleteAllCachedBiometrics();
+			logger.info("[Configuration updated] overrideFindDuplicate: " + proxyAbisConfigService.getDuplicate());
+			return new ResponseEntity<>("Successfully deleted cached biometrics", HttpStatus.OK);
+		} catch (RuntimeException exp) {
+			logger.error("Exception in cache request: " + exp.getMessage());
+			throw exp;
+		}
+	}
+
+	@RequestMapping(value = "checkDuplicate", method = RequestMethod.GET)
+	public ResponseEntity<String> setExpectation(@RequestParam("newduplicate") boolean newDuplicate) throws Exception {
+		logger.info("Setting new duplicate " + newDuplicate);
+		proxyAbisConfigService.setNewDuplicate(newDuplicate);
+		logger.info("Get new duplicate " + proxyAbisConfigService.getNewDuplicate());
+		return new ResponseEntity<>("Successfully updated " + proxyAbisConfigService.getNewDuplicate(), HttpStatus.OK);
+	}
+
+	@PostConstruct
+	private void postConstruct() {
```

Comment:
postconstruct should be in the service method since we are using this new duplicate in service itself.